evm/vm/forks/eip150/opcodes.py

```diff
@@ -2,7 +2,7 @@
 
 from cytoolz import merge
 
-from evm import constants
+from evm.vm.forks.eip150 import constants
```

Comment:
note to self: we should rename `forks.eip150` to be `forks.tangerine` since that's the official name of that fork.  Not something that should be done in this pull request.